control-plane/connect-inject/controllers/pod/pod_controller.go

```diff
@@ -903,35 +665,3 @@ func getUpstreamsID(name, namespace, partition string) *pbresource.ID {
 		},
 	}
 }
-
-func upstreamReferenceType() *pbresource.Type {
-	return &pbresource.Type{
-		Group:        "catalog",
-		GroupVersion: "v1alpha1",
-		Kind:         "Service",
-	}
-}
```

Comment:
Heads up, this will get deleted when https://github.com/hashicorp/consul-k8s/pull/2993 is merged, at which point you'll want to replace the new one too. Since it needs an export here, you're probably stuck w/ a conflict either way, so probably just a `TODO` comment.

Reply:
Thanks for the heads up, I'll also have to make the other changes made in that PR to these functions.